src/xn.ml

```diff
@@ -535,6 +535,21 @@ let diagnostics' () =
 	Client.get_diagnostics dbg () |> Jsonrpc.of_string |> pp;
 	`Ok ()
 
+let stat_vm _ id =
+  let open Vm in
+  let id = match id with | Some id -> id | None -> failwith "Need VM" in
```

Comment:
I don't this is the right way to handle this, because it throws an exception meaning something has gone unexpectedly wrong. I believe we should make use of the existing need_vm function which prints the user friendly message "xenops-cli: You must supply a VM name or UUID"